ops2.c

```diff
@@ -1734,6 +1734,31 @@ static void x86emuOp2_movsx_word_R_RM(x86emu_t *emu, u8 op2)
 }
 
 
+/****************************************************************************
+REMARKS:
+Handles opcode 0x0f,0xc8-0xcf
+****************************************************************************/
+static void x86emuOp2_bswap(x86emu_t *emu, u8 op2)
+{
+  u32 *reg32, tmp;
+
+  OP_DECODE("bswap ");
+
+  if(MODE_DATA32) {
+    reg32 = decode_rm_long_register(emu, op2 & 0x7);
+    tmp = 0;
+    tmp |= (*reg32 >> 24) & 0x000000ff;
+    tmp |= (*reg32 >>  8) & 0x0000ff00;
+    tmp |= (*reg32 <<  8) & 0x00ff0000;
+    tmp |= (*reg32 << 24) & 0xff000000;
+
+    *reg32 = tmp;
+  }
+  else {
+    INTR_RAISE_UD(emu);
```

Comment:
That's IMHO not correct. The spec says the outcome is 'undefined'; on my Intel processor it's just a NOP. I would
simply make the operation a NOP.

Reply:
Yes, that's true. Fixed.